.changelog/4535.yml

```diff
@@ -0,0 +1,4 @@
+changes:
+- description:  Moved RM102 and IN136 validation to the new format. Ensures context outputs in README and YML files match.
```

Comment:
Add another, breaking note (another item in the yml list), about the unification.